conans/client/generators/cmake_find_package_common.py

```diff
@@ -51,11 +55,15 @@
     list(APPEND {name}_LIBRARIES{build_type_suffix} ${{_FRAMEWORK}})
 endforeach()
 
-foreach(_SYSTEM_LIB ${{{name}_SYSTEM_LIB{build_type_suffix}}})
+foreach(_SYSTEM_LIB ${{{name}_SYSTEM_LIBS{build_type_suffix}}})
     list(APPEND {name}_LIBRARIES_TARGETS{build_type_suffix} ${{_SYSTEM_LIB}})
     list(APPEND {name}_LIBRARIES{build_type_suffix} ${{_SYSTEM_LIB}})
 endforeach()
 
+# We need to add our requirements too
+set({name}_LIBRARIES_TARGETS{build_type_suffix} "${{{name}_LIBRARIES_TARGETS{build_type_suffix}}};{deps_names}")
```

Comment:
Not sure why dependencies are being added to this variable, it seems defined for self package things, not for the transitive dependencies values.

Reply:
Maybe this is a misunderstanding on my side. I'm adding the dependencies because we are already adding the system_libs two lines above... and the consumer (outside Conan, in the user CMakeLists.txt) will need these libraries too in order to link with all the things needed.

Reply:
I thought the purpose of that variable is to have the current package things, not the transitive ones, so if the user wants to independently use somehow different things will be using ``BOOST_LIBRARIES_TARGETS`` and ``ZLIB_LIBRARIES_TARGETS``? This variable is really not documented in https://docs.conan.io/en/latest/reference/generators/cmake_find_package.html, so I guess not a big deal. The docs say that the targets are transitive, but doesn't say much about other variables.